gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/transferFeedOwnership.ts

```diff
@@ -38,6 +38,11 @@ export default class TransferFeedOwnership extends SolanaCommand {
         isSigner: false,
         isWritable: true,
       },
+      {
```

Comment:
Can you switch these two to use `const createIx = await program.instruction.transferFeedOwnership({ ..` like we do in the other commands (see createProposal for example)? We're trying to port the commands away from `program.coder.instruction.encode`